settings.gradle

```diff
@@ -108,6 +108,7 @@ include 'instrumentation:apache-pekko-http-2.13_1'
 include 'instrumentation:apache-pekko-http-3_1'
 include 'instrumentation:apache-pekko-http-core-2.13_1'
 include 'instrumentation:apache-struts-2.0'
+include 'instrumentation:apache-struts-2.0_6.7.0'
```

Comment:
any reason to not name this module `apache-struts-6.7.0`?

Reply:
Only because we named the original module `apache-struts-2.0` (which was my mistake).

I'm actually fine renaming this module and the original struts2 module if everyone else is ok with it.